rocAL/rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -200,38 +200,33 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalJpegCOCOFileSourcePartialSingleShard
                                                             unsigned shard_id,
```

Comment:
Please check the params list, it contains xdrift factor and ydrift factor which is not used in this API